modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -43,22 +47,22 @@ This process requires that you have write access to a container image registry o
 endif::restricted[]
 
 .Prerequisites
-ifndef::openshift-rosa,openshift-dedicated[]
+ifndef::openshift-rosa,openshift-dedicated,oc-mirror-v2[]
```

Comment:
Is there a reason why you wouldn't want to show this line for oc-mirror-v2?

This prereq does seem important to show for oc-mirror v2:

> * You configured a mirror registry to use in your disconnected environment.

Otherwise nothing shows that they have to have a mirror registry as a prereq. I think you need to remove the oc-mirror-v2 from this ifndef/endif pair